sqlstore/src/main/resources/db/migration_afterMigrate/90_targeted_sequencing.sql

```diff
@@ -54,8 +54,8 @@ CALL addTargetedSequencing('SureSelect v4','GSLE Legacy','GA_multiplex',1);
 CALL addTargetedSequencing('SureSelect v4','GSLE Legacy','KAPA Hyper Prep',1);
 CALL addTargetedSequencing('SureSelect v5', 'FFPE-R', 'Agilent SureSelect XT2', 0);
 CALL addTargetedSequencing('SureSelect v5','GSLE Legacy','KAPA Hyper Prep',1);
-CALL addTargetedSequencing('SureSelect v5','GSLE Legacy','QXT',0);
 CALL addTargetedSequencing('SureSelect v5','FFPE-R','KAPA Hyper Prep and SureSelect XT2 v5',0);
+CALL addTargetedSequencing('SureSelect v6','FNS','QXT',0);
```

Comment:
A link between 'Agilent SureSelect Human All Exon V6' and 'QXT' (the other entry below) likely does need created.

Do not create 'SureSelect v6' as it would be a duplicate of 'Agilent SureSelect Human All Exon V6'